packages/stopwatch/tsup.config.ts

```diff
@@ -1,3 +1,5 @@
 import { createTsupConfig } from '../../scripts/tsup.config';
 
-export default createTsupConfig({ format: ['esm', 'cjs'] });
+export default createTsupConfig({
+	globalName: 'Stopwatch'
```

Comment:
```suggestion
	globalName: 'SapphireStopwatch'
```

`Sapphire` prefix to retain consistency